cloudconfig/instancecfg/instancecfg.go

```diff
@@ -633,13 +633,9 @@ func (cfg *ControllerConfig) VerifyConfig() error {
 	return nil
 }
 
-// DefaultBridgePrefix is the prefix for all network bridge device
-// name used for LXC and KVM containers.
-const DefaultBridgePrefix = "br-"
-
 // DefaultBridgeName is the network bridge device name used for LXC and KVM
 // containers
-const DefaultBridgeName = DefaultBridgePrefix + "eth0"
+const DefaultBridgeName = "br-eth0"
```

Comment:
I think that code is actually a lie from back when we had juju-br0 and br0 and other such things before we did per-device bridging. I'll need to understand it a little bit better, looking.